tests/run_tests.sh

```diff
@@ -20,28 +20,19 @@ STATUS=0
 
 export HDF5_DISABLE_VERSION_CHECK=1
 
-# Run the standard set of unit tests, producing JUnit compatible output
-coverage run -m pytest tests --junitxml=tests/reports/test-results-"$(date +'%Y%m%d%H%M%S')".xml
-RESULT=$?
 
-if [ "$RESULT" -ne "0" ]; then
-    STATUS=1
-    echo "ERROR: unittest generated errors"
-fi
+# Exit status used to report back to caller
+STATUS=0
```

Comment:
Looking at the raw file
https://github.com/nasa/harmony-browse-image-generator/blob/8bf518b4a1af610f408b13fcb7d3e9356e44a01b/tests/run_tests.sh

Double declaration of STATUS=0 on Line 19 Line 25

Reply:
38d9f1b